lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -40,6 +40,14 @@ function subscribeEvents() {
     const label = args !== null ? '#' + args.attr('id') : '';
     ga('send', 'event', 'Modal', 'Close', label);
   });
+
+  // Signup button on pitch page
+  const urlParts = window.location.pathname.split('/');
+  if (urlParts[urlParts.length - 1] === 'pitch') {
```

Comment:
Have you tested this with a normal account? As far as I  know, `/pitch` is just an override URL for admins.

For example, this is the pitch page URL for Lose Your V-Card (for a logged-out or not signed up user):

[`https://www.dosomething.org/us/campaigns/lose-your-v-card`](https://www.dosomething.org/us/campaigns/lose-your-v-card)